### PR TITLE
stablize emoji related deps + fix missing emojis

### PR DESCRIPTION
**Pull request by mikedemarais**
this one emoji package i use was unstable, but now all the emojis should work

### CODE REVIEW

File: src/components/floating-emojis/ComingSoonFloatingEmojis.js [hunk 1]
```diff
@@ -2,7 +2,15 @@ import React from 'react';
 import FloatingEmojis from './FloatingEmojis';
 import FloatingEmojisTapHandler from './FloatingEmojisTapHandler';
 
-const emojis = ['soon', 'soon', 'soon', 'soon', 'unicorn', 'soon', 'rainbow'];
+const emojis = [
+  'soon_arrow',
```
Comment (brunobar79) [inline src/components/floating-emojis/ComingSoonFloatingEmojis.js:6]: The soon arrow isn't showing for me: http://recordit.co/od30bdeCqC

File: src/components/floating-emojis/FloatingEmojis.js [hunk 2]
```diff
@@ -136,9 +136,8 @@ FloatingEmojis.defaultProps = {
   duration: 2000,
   // Defaults the emoji to 👍️ (thumbs up).
   // To view complete list of emojis compatible with this component,
-  // head to https://unicodey.com/emoji-data/table.htm and reference the
-  // table's "Short Name" column.
-  emojis: ['+1'],
+  // head to https://github.com/muan/unicode-emoji-json/blob/master/data-by-emoji.jso
+  emojis: ['thumbs_up'],
```
Comment (brunobar79) [inline src/components/floating-emojis/FloatingEmojis.js:140]: This one isn't showing either: http://recordit.co/DWHJVzxtZq

File: src/components/settings-menu/SettingsSection.js [hunk 3]
```diff
@@ -194,7 +194,7 @@ const SettingsSection = ({
           onPress={onSendFeedback}
         />
         <ListItem
-          icon={<Emoji name="heart" />}
+          icon={<Emoji name="red_heart" />}
```
Comment (brunobar79) [inline src/components/settings-menu/SettingsSection.js:197]: Screen Shot 2020-08-25 at 11 35 46 AM

File: src/components/coin-row/ExchangeCoinRow.js [hunk 4]
```diff
@@ -22,7 +22,7 @@ const FloatingFavoriteEmojis = styled(FloatingEmojis).attrs({
   disableVerticalMovement: true,
   distance: 70,
   duration: 400,
-  emojis: ['star2'],
+  emojis: ['glowing_star'],
```
Comment (brunobar79) [inline src/components/coin-row/ExchangeCoinRow.js:25]: Not showing up

**Comment by mikedemarais** [comment]
hey @brunobar79 i believe this commit fixes all these problems. assuming bitrise passes i think this PR is good to merge https://github.com/rainbow-me/rainbow/pull/1027/commits/5a70729c843d8e84b1ad1425a67f19945c6de16a

**Pull request merged**